### ISSUE TITLE
Validate that the vpc-cni addon created for an IPv6 cluster is at least 1.10.0

### ISSUE THREAD

**Issue by nikimanoledaki** [weaveworks/eksctl#4463]
We need to add some validation that the version of the `vpc-cni` addon created for an IPv6 cluster is at least `1.10.0`, which is `latest` at the time of writing this issue. 

The current default version for `vpc-cni` is lower than `1.10.0` and it throws an error during cluster creation.

**Comment by aclevername** [comment]
is this something we want to do ourselves? I'm wary of adding this in as it might fall out of date quickly :thinking: given the user already gets an error during the `eksctl create cluster`, its not terrible UX atm. A trade-off for sure

**Comment by nikimanoledaki** [comment]
That's true and I'm glad you raised this - we do have validation for the k8s version being at least `1.21` that will also be outdated eventually, so maybe we need to find a way to track these kinds of things?

Another thing is that the cluster creation failed with this error after maybe 15 minutes so it would be nice if eksctl could fail fast.

**Comment by aclevername** [comment]
> That's true and I'm glad you raised this - we do have validation for the k8s version being at least `1.21` that will also be outdated eventually, so maybe we need to find a way to track these kinds of things?

I don't think things like that will get out of date, they will never retroactively add support to older versions of k8s. Your right though that its in a similar vain- we hard code things in that *could* eventually be wrong

> Another thing is that the cluster creation failed with this error after maybe 15 minutes so it would be nice if eksctl could fail fast.

Yeah I do agree, but its also not the worst. I'm not opposed to have this version check in our code, but its just something to consider the cost. Could we just document ipv6 to always have `version: latest` for the addon and that be enough?

**Comment by nikimanoledaki** [comment]
Good points all around! How about we wait for planning to hear what other people think?

**Comment by Himangini** [comment]
We might revisit this later to access if the costs are getting higher to maintain this.
